stdlib/distutils/command/install.pyi

```diff
@@ -4,8 +4,8 @@ from typing import Any, ClassVar, Final
 
 from ..cmd import Command
 
-HAS_USER_SITE: bool
-SCHEME_KEYS: tuple[str, ...]
+HAS_USER_SITE: Final[bool]
+SCHEME_KEYS: Final[tuple[purelib, platlib, headers, scripts, data]]
```

Comment:
My bad, these should be `Literal[]`, that's what I get for writing the suggestion from memory.

 ```suggestion
from typing import Literal
SCHEME_KEYS: Final[tuple[Literal["purelib"], Literal["platlib"], Literal["headers"], Literal["scripts"], Literal["data"]]]
```